scripts/mmuconfig

```diff
@@ -196,9 +196,9 @@ def main(show_config, profile, alpha, staticth, trim, namespace, verbose):
             mmu_cfg.list()
         # Buffershow cannot modify profiles
         elif config and profile:
-            if alpha:
+            if alpha is not None:
                 mmu_cfg.set(profile, DYNAMIC_THRESHOLD, alpha)
-            elif staticth:
+            elif staticth is not None:
```

Comment:
>staticth [](http://example.com/codeflow?start=17&length=8)

If `alpha is not None`, why you skip `staticth` checking branch? I think they should be independent with each other.

Reply:
@qiluo-msft static th is for static threshold. Dynamic threshold is for dynamic threshold 
Imo a buffer profile cannot be both dynamic and static at the same time.

Reply:
Same question to trim.

Reply:
@qiluo-msft trim is the legacy logic. I didn't change them.